src/find/matchers/name.rs

```diff
@@ -47,15 +47,19 @@ mod tests {
     fn create_file_link() {
         #[cfg(unix)]
         if let Err(e) = symlink("abbbc", "test_data/links/link-f") {
-            if e.kind() != ErrorKind::AlreadyExists {
-                panic!("Failed to create sym link: {:?}", e);
-            }
+            assert!(
+                !(e.kind() != ErrorKind::AlreadyExists),
```

Comment:
There's a double negation here now that can be removed:
```suggestion
                e.kind() == ErrorKind::AlreadyExists,
```
Clippy probably would suggest the same thing.

Reply:
implementing it